diff --git a/dist/node/chunks/dep-Chhhsdoe.js b/dist/node/chunks/dep-Chhhsdoe.js
index 0000000000000000000000000000000000000000..0000000000000000000000000000000000000000 100644
--- a/dist/node/chunks/dep-Chhhsdoe.js
+++ b/dist/node/chunks/dep-Chhhsdoe.js
@@ -2414,6 +2414,12 @@ function optimizeSafeRealPathSync() {
 			return;
 		}
 	}
+	try {
+		exec("net use", (error$1, stdout) => {
+			if (error$1) return;
+			const lines = stdout.split("\n");
+			for (const line of lines) {
+				const m$2 = parseNetUseRE.exec(line);
+				if (m$2) windowsNetworkMap.set(m$2[2], m$2[1]);
+			}
+			if (windowsNetworkMap.size === 0) safeRealpathSync = fs.realpathSync.native;
+			else safeRealpathSync = windowsMappedRealpathSync;
+		});
+	} catch {
+		safeRealpathSync = fs.realpathSync.native;
+	}
+	return;
 	exec("net use", (error$1, stdout) => {
 		if (error$1) return;
 		const lines = stdout.split("\n");
